Doc/library/sqlite3.rst

```diff
@@ -1480,6 +1479,33 @@ regardless of the value of :attr:`~Connection.isolation_level`.
    https://www.sqlite.org/lang_transaction.html#deferred_immediate_and_exclusive_transactions
 
 
+.. _sqlite3-uri-tricks:
+
+SQLite URI tricks
+-----------------
+
+Some useful URI tricks include::
+
+   # Open a database in read-only mode.
+   con = sqlite3.connect("file:template.db?mode=ro", uri=True)
+
+   # Don't implicitly create a new database file if it does not already exist.
+   # Will raise sqlite3.OperationalError if unable to open a database file.
+   con = sqlite3.connect("file:nosuchdb.db?mode=rw", uri=True)
+
+   # Create a shared named in-memory database.
+   con1 = sqlite3.connect("file:mem1?mode=memory&cache=shared", uri=True)
+   con2 = sqlite3.connect("file:mem1?mode=memory&cache=shared", uri=True)
+   con1.execute("create table t(t)")
+   con1.execute("insert into t values(28)")
+   con1.commit()
+   rows = con2.execute("select * from t").fetchall()
```

Comment:
```suggestion
Some useful URI tricks include:

* Open a database in read-only mode::

      con = sqlite3.connect("file:template.db?mode=ro", uri=True)

  * Don't implicitly create a new database file if it does not already exist;
    will raise :exc:`~sqlite3.OperationalError` if unable to open one::
  
        con = sqlite3.connect("file:nosuchdb.db?mode=rw", uri=True)

  * Create a shared named in-memory database::
  
      con1 = sqlite3.connect("file:mem1?mode=memory&cache=shared", uri=True)
      con2 = sqlite3.connect("file:mem1?mode=memory&cache=shared", uri=True)
      con1.execute("create table t(t)")
      con1.execute("insert into t values(28)")
      con1.commit()
      rows = con2.execute("select * from t").fetchall()
```

Instead of keeping everything in one big verbatim code block, what about breaking this out into a bulleted list with each specific subtask, since each is self-contained and serves a different specific purpose?